### DISCUSSION TITLE
全部都跳过 有什么办法解决吗

### DISCUSSION THREAD

**Discussion by YBYCS** [Mufanc/iSmartAuto#24] [Q&A]
![image](https://user-images.githubusercontent.com/87234506/170861693-a0c95334-8347-4068-852e-fc7f2bbe915a.png)

**Comment by 5gentle5** [comment]
> ![image](https://user-images.githubusercontent.com/87234506/170861693-a0c95334-8347-4068-852e-fc7f2bbe915a.png)
> 
知道了，敲我下，谢谢。

**Comment by uboolean** [comment]
同求，知道了也敲我一下

**Comment by learngodie** [comment]
我也这样

**Comment by Javazzx** [comment]
![image](https://user-images.githubusercontent.com/104267380/172362870-fd4f012e-0de9-4e93-9389-ba2557c34366.png)

**Comment by zdlsya** [comment]
我也是这样，知道的敲我，谢谢
